### PR TITLE
Vj sfu cog

### PR DESCRIPTION
**Pull request by TitanVJ**
# SFU commands are here
Praise

## Added sfu command
* It returns calendar description from the current semester of the arg
* Usage: `.sfu <arg>`
    * Ex: `.sfu cmpt300`

## Added outline command
* It returns outline detail of the arg, optionally you can specify term and/or section
* Usage: `.outline <arg0> [<arg1> <arg2>]`
    * Ex: `.outline cmpt300 spring d200`

## All the thing have been updated to go along with these commands.
* Entries in : 
    * help.json
    * cogs.json
    * README.md
    * My soul
## Test cases
* SFU command
    * `.sfu cmpt 300`
    * `.sfu cmpt300`
    * `.sfu cmpt666`
    * `.sfu blah`
    * `.sfu`
* Outline command
    * `.outline cmpt300`
    * `.outline cmpt 300`
    * `.outline cmpt300 spring d200`
    * `.outline cmpt 300 spring d200`
    * `.outline cmpt666`
    * `.outline blah`
    * `.outline`

### CODE REVIEW

**Comment by TitanVJ** [comment]
@modernNeo done

**Comment by modernNeo** [comment]
@TitanVJ i know you/discord has waited a while but I will read over the code and approve tmr, i got homework today

**Comment by TitanVJ** [comment]
ok

**Comment by TitanVJ** [comment]
@modernNeo done

File: commands_to_load/SFU.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+        url = 'http://www.sfu.ca/bin/wcm/academic-calendar?%s/%s/courses/%s/%s' % (year, term, courseCode, courseNum)
+        logger.info('[SFU sfu()] url for get request constructed: %s' % url)
+
+        res = req.get(url)
+        if(res.status_code == 200):
+            logger.info('[SFU sfu()] get request successful')
+            data = res.json()
+        else:
+            logger.info('[SFU sfu()] get resulted in 404')
```
Comment (modernNeo) [inline commands_to_load/SFU.py]: can you guarantee that the error will always be 404?

Comment (TitanVJ) [reply commands_to_load/SFU.py]: done. changed for both outline and sfu commands

Comment (modernNeo) [reply commands_to_load/SFU.py]: you sure? cause its still in the logger.info....

File: commands_to_load/SFU.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        # Set up url for get
+        url = 'http://www.sfu.ca/bin/wcm/course-outlines?%s/%s/%s/%s/%s' % (year, term, courseCode, courseNum, section)
+        logger.info('[SFU outline()] url for get constructed: ' + url)
+
+        res = req.get(url)
+        if(res.status_code == 200):
+            logger.info('[SFU outline()] get request successful')
+            data = res.json()
+        else:
+            logger.error('[SFU outline()] get resulted in 404')
```
Comment (modernNeo) [inline commands_to_load/SFU.py]: i dont see it updated

**Comment by TitanVJ** [comment]
should be working now. forgot to pull the updates before pushing lol

**Comment by TitanVJ** [comment]
@wnfldchen will work on tonight. ill update urban as well but in its own pr.

**Comment by TitanVJ** [comment]
done converting to async get's @wnfldchen

**Pull request merged**